homeassistant/components/nextdns/config_flow.py

```diff
@@ -128,3 +128,31 @@ async def async_step_reauth_confirm(
             data_schema=AUTH_SCHEMA,
             errors=errors,
         )
+
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle a reconfiguration flow initialized by the user."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
```

Comment:
The code is almost identical with `reauth`. It could be combined into a separate function and called from the respective step to make the code a bit more dry.